assets/app/util/githubApi.js

```diff
@@ -1,25 +1,31 @@
 import http from 'axios';
+import fetch from './fetch';
 
 import store from '../store';
 import { encodeB64, decodeB64 } from './encoding';
 import errorActions from '../actions/errorActions';
 
 const API = 'https://api.github.com';
 
+// Thinking this function should not live here
+// Service should be agnostic as to what the store is.
+// Access token should be passed in to the functions
```

Comment:
Thats what I was thinking too, just store the token on instantiation. This only happens once per client session, provided the object gets initialized after the user logs in, and cleaned up when they log out.